Makefile

```
@@ -297,7 +297,7 @@ sq = $(subst ','\'',$(1)) # function to escape single quotes (')
 TAGS:
 	etags stbt_core/**.py _stbt/**.py
 
-_stbt/libstbt.so : _stbt/sqdiff.c
+_stbt/libstbt.x86_64.so : _stbt/sqdiff.c
```

```suggestion
PLATFORM:=$(shell python3 -c 'import platform; print(platform.machine())')
_stbt/libstbt.$(PLATFORM).so : _stbt/sqdiff.c
```

Or maybe just hard-code `PLATFORM:=x86_64` here, but at least people could manually override it.